<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadwidget</class>
 <widget class="QWidget" name="downloadwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(45, 45, 45);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>244</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_file">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
        </property>
        <property name="icon">
         <iconset resource="lib.qrc">
          <normaloff>:/lib/download.png</normaloff>:/lib/download.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_body" native="true">
     <property name="styleSheet">
      <string notr="true">#widget_body{
background-color: rgb(162, 162, 162);
/*padding:5px;*/
border-radius:6px;
border:2px solid rgb(108, 117, 125, 65); 
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(125, 125, 125, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>OPPOSans B</family>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>当前下载路径： C:\Users\张旭\Desktop\fsdownload</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_file_3">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
           </property>
           <property name="icon">
            <iconset resource="lib.qrc">
             <normaloff>:/lib/file.png</normaloff>:/lib/file.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_file_2">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgba(255, 255, 255, 0);
	border: none;
}</string>
           </property>
           <property name="icon">
            <iconset resource="lib.qrc">
             <normaloff>:/lib/setting2.png</normaloff>:/lib/setting2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">#scrollArea{
	border: 1px solid;
	background-color: rgb(29, 29, 29);
}
</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>268</width>
           <height>238</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 150);</string>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>251</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>9</pointsize>
               <stylestrategy>PreferAntialias</stylestrategy>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(45, 45, 45);</string>
             </property>
             <property name="text">
              <string>bison-2.5.1.tar.gz</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QProgressBar" name="progressBar">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>241</width>
            <height>18</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QProgressBar {
	border: 1px solid;
	border-color: rgb(91, 91, 91);
	border-radius: 0px; 
	/*background-color: #FFFFFF; */
	text-align:center; 
	color: rgba(0, 0, 0, 200);
}

QProgressBar:chunk{
	border-radius:0px;
	background-color: rgb(180, 220, 255);
}</string>
          </property>
          <property name="value">
           <number>95</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="textDirection">
           <enum>QProgressBar::TopToBottom</enum>
          </property>
          <property name="format">
           <string>%p%</string>
          </property>
         </widget>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>70</y>
            <width>241</width>
            <height>18</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(131, 185, 255, 0);
color: rgb(0, 0, 0);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>3</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="minimumSize">
              <size>
               <width>12</width>
               <height>12</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>12</width>
               <height>12</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/lib/peixu.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>99</width>
               <height>15</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>15</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(45, 45, 45);</string>
             </property>
             <property name="text">
              <string>52.6% 456k/s  23M</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
